integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/search/query/ElasticsearchQueryStringPredicateSpecificsIT.java

```diff
@@ -0,0 +1,129 @@
+import org.hibernate.search.util.impl.integrationtest.mapper.stub.BulkIndexer;
+import org.hibernate.search.util.impl.integrationtest.mapper.stub.SimpleMappedIndex;
+import org.hibernate.search.util.impl.integrationtest.mapper.stub.StubMappingScope;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+class ElasticsearchQueryStringPredicateSpecificsIT {
+
+	protected static final String DOCUMENT_1 = "document1";
+	protected static final String DOCUMENT_2 = "document2";
+
+	protected static final String TEXT_1 = "Here I was, feeding my panda, and the crowd had no word.";
+	protected static final String TEXT_2 = "Without a word, he went out of the room.";
+
+	@RegisterExtension
+	public static final SearchSetupHelper setupHelper = SearchSetupHelper.create();
+
+	protected static final SimpleMappedIndex<IndexBinding> index = SimpleMappedIndex.of( IndexBinding::new );
+
+	private static final DataSet dataSet = new DataSet();
+
+	@BeforeAll
+	static void setup() {
+		setupHelper.start()
+				.withBackendProperty(
+						ElasticsearchIndexSettings.ANALYSIS_CONFIGURER,
+						(ElasticsearchAnalysisConfigurer) context -> context.analyzer( "with-stop-filter" )
+								.custom()
+								.tokenizer( "standard" )
+								.tokenFilters( "lowercase", "stop" )
+				)
+				.withIndexes( index )
+				.setup();
+
+		BulkIndexer indexer = index.bulkIndexer();
+		dataSet.contribute( indexer );
+		indexer.join();
+	}
+
+	@Test
+	void enablePositionIncrements_disabled() {
+		StubMappingScope scope = index.createScope();
+		String absoluteFieldPath = index.binding().analyzedStringField1.relativeFieldName;
+
+		SearchQuery<DocumentReference> query = scope.query()
+				.where( f -> f.queryString()
+						.field( absoluteFieldPath )
+						.matching( "\"out of the room\"" )
+						.enablePositionIncrements( false ) )
+				.toQuery();
+
+		assertThatQuery( query )
+				.hasNoHits();
+	}
+
+	@Test
+	void enablePositionIncrements_enabled() {
+		StubMappingScope scope = index.createScope();
+		String absoluteFieldPath = index.binding().analyzedStringField1.relativeFieldName;
+
+		SearchQuery<DocumentReference> query = scope.query()
+				.where( f -> f.queryString()
+						.field( absoluteFieldPath )
+						.matching( "\"out of the room\"" )
+						.enablePositionIncrements( true ) )
+				.toQuery();
+
+		assertThatQuery( query )
+				.hasDocRefHitsAnyOrder( index.typeName(), DOCUMENT_2 );
+	}
+
+	@Test
+	void enablePositionIncrements_default() {
+		StubMappingScope scope = index.createScope();
+		String absoluteFieldPath = index.binding().analyzedStringField1.relativeFieldName;
+
+		SearchQuery<DocumentReference> query = scope.query()
+				.where( f -> f.queryString()
+						.field( absoluteFieldPath )
+						.matching( "\"out of the room\"" ) )
+				.toQuery();
+
+		assertThatQuery( query )
+				.hasDocRefHitsAnyOrder( index.typeName(), DOCUMENT_2 );
+	}
```

Comment:
Why is this specific to Elasticsearch?

If it's just about the analyzer definition, you can make it backend-independent; see:

https://github.com/hibernate/hibernate-search/blob/6354c347a466be9f8a04d4bf985736b3bcdf68cc/integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/analysis/AnalysisCustomIT.java#L259

Reply:
oh nice, yeah, only beacuse of the analyzer, to make some meaningful test for this option there should be a stop filter 🙈, thanks for the pointer 😃 👍🏻